bcml/cli.py

```diff
@@ -0,0 +1,184 @@
+                meta=meta,
+            )
+
+        else:
+            # https://stackoverflow.com/questions/24264609/parsing-txt-file-into-dictionary-in-python
+            meta = {}
+            with info.open(encoding="utf-8") as rules:
+                for line in rules:
+                    if "=" in line:
+                        key, val = map(str.strip, line.split("="))
+                        meta[key] = val
+            dev.create_bnp_mod(
+                mod=args.bnpify,
+                output=args.bnpify / f"{meta['name'].replace(' ', '')}.bnp",
+                meta={
+                    "name": meta["name"],
+                    "version": meta["version"],
+                    "desc": meta["description"],
+                    "platform": "wiiu"
+                    if get_settings("wiiu")
+                    else "switch",
+                },
+            )
+    elif args.export:
+        # --export
+        install.export(args.export)
+
+    elif args.standalone:
+        # --standalone
+        bnp = args.standalone
+        with TempModContext():
+            install.install_mod(
+                bnp,
+                merge_now=True,
+                options={"options": {"texts": {"all_langs": True}}, "disable": []},
+            )
+            
+            out = f"{bnp.stem}.zip"
+            if out:
+                output = Path(out)
+                install.export(output, standalone=True)
+    
+    elif args.update:
+        # --update
+        update_file = args.update;
+        for i in get_installed_mods(disabled=True):
+            name = i.name.replace(" ", "").lower()
+            arg = update_file.name.rsplit(".")[0].replace(" ", "").lower()
+            if name == arg:
+                mod = i;
+
+        if (mod.path / "options.json").exists():
+            options = json.loads((mod.path / "options.json").read_text());
+        else:
+            options = {};
+
+        shutil.rmtree(mod.path)
+        with Pool(maxtasksperchild=500) as pool:
+            new_mod = install.install_mod(
+                Path(update_file),
+                insert_priority=mod.priority,
+                options=options,
+                pool=pool,
+                updated=True,
+            );
+
+        install.refresh_master_export()
+
+    elif args.remerge:
+        # --remerge
+        install.refresh_merges()
+        install.refresh_master_export()
+
+    elif args.install:
+        # --install
+        install.install_mod(
+            mod=args.install, 
+            insert_priority=args.priority if args.priority else 0, 
+            merge_now=True
+        )
+        install.refresh_master_export()
+
+    elif args.uninstall:
+        # --uninstall
+        for mod in util.get_installed_mods(disabled=True):
+            name = mod.name.replace(" ", "").lower()
+            arg = args.uninstall.replace(" ", "").lower()
+            if name.lower() == arg.lower():
```

Comment:
They are already `.lower()`ed from their assignment. This won't break anything, nor will it make a perceptible speed difference, but there's not really a reason for it, either

Reply:
Oops, sorry about that. I removed the `.lower()` call and the whitespace edit